src/lib/pickles/wrap_verifier.ml

```diff
@@ -432,33 +444,45 @@ struct
     end
 
     let combine batch ~xi without_bound with_bound =
+      let reduce_point p =
```

Comment:
I dream to have tests verifying the number of constraints checking the changes are following the spec  :).

Reply:
Could be simly regression tests for the moment, where we dump the number of constraints, and we verify if it increases/decreases when the code changes.

Reply:
This now exists in `develop`.